docs/ota-client-guide/modules/ROOT/pages/device-monitoring-with-zabbix.adoc

```diff
@@ -0,0 +1,227 @@
+
+=== Table of contents
+* <<Install and setup Zabbix server on Ubuntu 16/18>>
+* <<Build an OTA image for QEMU>>
+* <<Run a QEMU image>>
+* <<Run a QEMU image in the background mode>> 
+* <<Add a template and configure a host for Zabbix server>>
```

Comment:
This isn't needed--Antora will automatically add a table of contents in the sidebar.